Dockerfile

```diff
@@ -1,15 +0,0 @@
-  

```

Comment:
We still need this file.  Keeping a Dockerfile at the root of a repo is pretty standard.